presto-main/src/main/java/com/facebook/presto/execution/scheduler/TableWriteInfo.java

```diff
@@ -161,12 +162,15 @@ private static Optional<DeleteScanInfo> createDeleteScanInfo(PlanNode planNode,
 
     private static Optional<DeleteScanInfo> createDeleteScanInfo(DeleteNode delete, Optional<ExecutionWriterTarget> writerTarget, Metadata metadata, Session session)
     {
-        TableHandle tableHandle = ((ExecutionWriterTarget.DeleteHandle) writerTarget.get()).getHandle();
+        /* Metadata.getLayout does not accept DeleteTableHandle
+        *  DeleteScanInfo should stay with TableHandle type so the effect does not continue to ripple out
+         */
+        DeleteTableHandle tableHandle = ((ExecutionWriterTarget.DeleteHandle) writerTarget.get()).getHandle(); //unused in placeholder fix
```

Comment:
Since this is unused, I think you can remove it and just have a clear comment that the original TableScanNode TableHandle is used.

Reply:
agreed, just remove this unsued handle.  Additionally, if you look at how this tableHandle from delete was created vs. the tableHandle from the delete tablescan, I believe would have been the same, so the change below is actually not changing the TableLayoutResult.